Library/Formula/get-flash-videos.rb

```diff
@@ -0,0 +1,16 @@
+class GetFlashVideos < Formula
+  homepage "https://code.google.com/p/get-flash-videos"
+  url "http://get-flash-videos.googlecode.com/files/combined-get_flash_videos-1.24", :using => :nounzip
+  version "1.24"
+  sha256 "e41da715f817bfa1a65ae82cc4f5a82268b6e72a7f4d90ffabf89b3a522cbb91"
+
+  def install
+    mv "combined-get_flash_videos-1.24", "get_flash_videos"
```

Comment:
I knew there had to be a way! I took forever trying to figure out how before giving up.